src/rebar_api.erl

```diff
@@ -29,3 +30,5 @@ info(Str, Args) -> ?INFO(Str, Args).
 warn(Str, Args) -> ?WARN(Str, Args).
 %% @doc logs with severity `error'
 error(Str, Args) -> ?ERROR(Str, Args).
+%% @doc raises an exception that will be handled by format_error/1
+raise(Term) -> ?PRV_ERROR(Term).
```

Comment:
This doesn't make sense. `?PRV_ERROR` is a macro because it uses `?MODULE` to set the module that `format_error/1` will be called in.